packages/sync/integration_test/test/subscriptions.test.js

```diff
@@ -0,0 +1,126 @@
+import { createClient } from '../../dist';
+import { TestStore } from '../utils/testStore';
+import { ToggleableNetworkStatus } from '../utils/network';
+import server from '../utils/server';
+import {
+  ADD_TASK,
+  GET_TASKS,
+  UPDATE_TASK_CONFLICT_REJECT,
+  UPDATE_TASK_CLIENT_RESOLUTION,
+  UPDATE_TASK_CUSTOM_CLIENT_RESOLUTION,
+  UPDATE_TASK_SERVER_RESOLUTION,
+  TASK_CREATED
+} from '../utils/graphql.queries';
+
+const newNetworkStatus = (online = true) => {
+  const networkStatus = new ToggleableNetworkStatus();
+  networkStatus.setOnline(online);
+  return networkStatus;
+};
+
+const customResolution = (serverData, clientData) => {
+  return {
+    ...serverData,
+    ...clientData,
+    title: 'custom'
+  }
+};
+
+const newClient = async (clientOptions = {}) => {
+  const config = {
+    httpUrl: "http://localhost:4000/graphql",
+    wsUrl: "ws://localhost:4000/graphql",
+    ...clientOptions,
+    conflictStrategy: {
```

Comment:
Are we using any conflict stuff for subscriptions?

Reply:
no, I've copy pasted the config, it's not needed, will remove it